config/ingestCalibs.py

```diff
@@ -14,7 +16,6 @@
                             }
 config.register.columns = {'filter': 'text',
                            'ccdnum': 'int',
-                           'path': 'text',
                            'calibDate': 'text',
```

Comment:
If the decision is to continue using `path` in the butler policy template of `defect` or/and `fringe`, I don't think `path` can be removed from the schema.